main.py

```diff
@@ -109,52 +109,47 @@ def slideshow(db, start, end, seconds="0.25", rand=False):
     try:
         for x in r:
             pokemon = db.get_pokemon(x)
-            scripter.change_terminal(pokemon)
+            scripter.change_terminal(pokemon.get_path())
             time.sleep(delay)
     except KeyboardInterrupt:
         print("Program was terminated.")
         sys.exit()
 
 
-def change_terminal_background(db, arg):
-    # Change the terminal background to the specified Pokemon, if it exists.
+def change_terminal_background(db, arg):  # arg is a pokemon_name
+    """Change the terminal background to the specified Pokemon, if it exists."""
     if arg in db:
         pokemon = db.get_pokemon(arg)
-        scripter.change_terminal(pokemon)
+        scripter.change_terminal(pokemon.get_path())
     else:  # If not found in the database, try to give suggestions.
         suggestions = db.names_with_infix(arg)
-        if len(suggestions) == 0:
+        try:
+            pokemon = suggestions[0]
+            scripter.change_terminal(pokemon.get_path())
+            print("Did you mean {}?".format(pokemon.get_name().title()))
+            if suggestions[1:]:
+                print("Other suggestions:")
+                print_columns(suggestions[1:])
+        except IndexError:
             print("No such Pokemon was found and no suggestions are available.")
-        elif len(suggestions) == 1:
-            scripter.change_terminal(suggestions[0])
-            print("Did you mean " + suggestions[0].get_name().capitalize() + "?")
-            scripter.change_terminal(suggestions[0])
-        else:
-            print("Did you mean " + suggestions[0].get_name().capitalize() + "?")
-            print("Other suggestions:")
-            print_columns(suggestions[1:])
-            scripter.change_terminal(suggestions[0])
 
 
-def change_wallpaper(db, arg):
-    # Change the wallpaper to the specified Pokemon, if it exists.
+def change_wallpaper(db, arg):  # arg is a pokemon_name
```

Comment:
I think all the other functions in this area call it arg so I did not want to change it for reasons of consistency.  I merely wanted to document the usage pattern.  I believe that it would actually work if arg was a valid pokemon id (str or int).